### DISCUSSION TITLE
Apple Music 還是 Spotify?

### DISCUSSION THREAD

**Discussion by kzx-creat** [1101-GS4719-C/Python-Community-GS4719-C#3392] [General]
滑IG看到別人推薦音樂，我發現幾乎都是從Spotify分享的，不知道大家聽音樂的管道是不是Spotify居多? 我目前是用Apple Music，很久很久之前有用過Spotify免費版的。

**Comment by christaaah** [comment]
我都用yt music，因為我有訂閱youtube premium，我覺得還蠻好用的，整體的使用非常的流暢，會推薦平常會廳的類型的音樂，也可以嘗試沒有接觸到的音樂，介面也做得蠻好看的。

**Comment by DarsooK** [comment]
我使用Youtube Music網頁版搭配Google Chrome擴充功能：Ad Block，使用Youtube上按讚喜歡的音樂，亦或是建立撥放清單，都能連結至Youtube Music，最重要的是使用Ad Block搭配Youtube再背景撥放、切換音樂時電腦會比較卡，搭配Youtube Music則比較順暢。

**Comment by ed112601513** [comment]
雖然我是蘋果用戶但是我沒用過Apple music，以後可以用看看，我覺得Spotify免費版就夠我用了，廣告沒很多，而有些人提到的Youtube Music，Youtube的東西如果不買premium廣告會超多，我又不敢用廣告攔截程式，畢竟之前Youtube一直放話或警告使用廣告攔截程式的人，所以我還是乖乖用Spotify免費版就好

**Comment by Elaine4488** [comment]
我是用Spotify免費版，雖然會有廣告，但我覺得我已經很夠用了，也用的蠻習慣的。而且還有分享歌單的功能，覺得還不錯，不確定Apple Music有沒有這個功能。目前沒有使用過Apple Music，或許之後可以來試用看看。

**Comment by qiu0109** [comment]
我沒有用過Apple Music，畢竟我有spotify的家庭方案可以使用，我覺得spotify使用很流暢，推薦的歌曲也都是我會喜歡的，所以我覺得聽音樂spotify是個很不錯的選擇。

**Comment by Mminniehuang** [comment]
我也是用youtube premium。
我沒有用過Apple Music，但我用過spotify免費版和一般的youtube，但有廣告還是讓人覺得有點煩，還有時候會突然被嚇到..
後來考量到畢竟除了聽歌還會看很多影片等等，所以選擇用youtube，也幾乎沒有在用spotify了。

**Comment by poyen0806** [comment]
我都用youtube music，因為有朋友買youtube premium分享給我，這樣我在消除廣告的同時，還能享受youtube的音樂，而且youtube的音樂演算法很懂我喜歡聽的歌，所以我就從spotify轉到youtube music了。不過沒有使用過Apple Music，或許之後可以來試用看看。

**Comment by Taoyi-Chiang** [comment]
我都用Youtube premium，很久以前有用過Apple music。我覺得Apple music的介面設計和路徑邏輯有點複雜，但最近他們有KTV系統，感覺滿好用的，一直在猶豫是不是可以跳回去Apple music。

**Comment by kai1660** [comment]
我使用Spotify，我使用免費版，有甚麼歌就聽甚麼，因為是免費的不需要要求這麼多，放一個聲音就好，也可以自己唱，但這樣有一個缺點是大家都聽得道，可能會吵到別人，所以也可以暢在心裡。

**Comment by vu45tp6** [comment]
我用apple music但是操作直覺上我覺得spotify更好一點

**Comment by ichenx2** [comment]
雖然我也是Apple用戶，但我平常聽音樂還是用Spotify居多，首先是我覺得Apple Music的UI設計上沒有那麼的直覺，沒有像Spotify一進去就推薦歌單給用戶。還有Apple Music會需要付費使用，不像Spotify有完全免費的版本可以使用，雖然有廣告，但其實好像也沒有那麼影響，因為廣告的長度其實就大概10秒~30秒左右，我覺得不太會影響聆聽體驗。另外，手機版的Spotify，如果是免費用戶的話，是沒辦法自行挑選音樂的播放順序的，但因爲我本人有嚴重的選擇障礙，所以這樣反而對我來說是好事，有人可以直接幫我決定歌單的順序。如果真的很想要自己安排歌曲的播放順序的話，可以考慮使用網頁版的Spotify，就可以自己隨意選擇播放順序！

**Comment by lilipop6405** [comment]
我覺得apple music只能在apple的產品上使用而已，不像spotify可以在很多平台上使用。就像我使用iphone手機和asus的電腦，兩個裝置都可以下載，非常方便!

**Comment by yurukung** [comment]
我用過Spotify 跟Apple music，我覺得各有優點，Spotify的演算法很強，可以讓我聽到很多我喜歡的風格，Apple Music的音質真的很好，但演算法可能就沒有這麼強（我自己的經驗）

**Comment by ZorasWu** [comment]
個人推薦spotify 我覺得apple music並不如預期的好用 而且他也有它的限制

**Comment by Kaz9487** [comment]
二選一的話，我個人是用Spotify，但Spotify上比youtube 少了很多音樂，對我來說只能拿來聽日文歌。這可能跟我自己比較喜歡聽古風歌有關，忘川風華錄的歌Spotify都沒有

**Comment by jaychang1027** [comment]
我自己是用spotify，但有蠻多歌在spotify上是找不到的，所以對我來說最好的選擇應該是youtube premiuma，但考慮到價格面向，還是覺得用spotify就好，而且我跟我室友會在宿舍開Jam一起點歌，就覺得這功能還不錯也用得到

**Comment by yunnnnchieh** [comment]
我自己是用apple music，介面上覺得使用起來很流暢，也有杜比音訊支援，有一個唱歌模式可以降低音樂的人聲讓你唱歌很酷，也可以在上面加朋友，可以得到一些朋友在聽的歌的推薦 很不錯，再加上新增在歌單的歌也都在裡面，所以也沒有想過要換

**Comment by 12388833** [comment]
Apple Music 還是 Spotify，其實真的看個人習慣。Apple Music 適合已經用 iPhone 或 Mac 的人，整合得很順，音質也很好，而且歌比較齊全，尤其是華語歌；但如果你喜歡發掘新歌、自己排播放清單，Spotify 的推薦系統真的很強，也比較有社群互動感。兩個平台都有學生方案和離線播放功能，功能差不多，就看你偏好介面、音樂風格還是哪個生態系。其實不妨都試用看看，找到最合自己才是重點。

**Comment by zhen-yuu** [comment]
Spotify用戶舉手，覺得他的分類模式蠻棒的，但是隨機播放的時候感覺沒有真的隨機，都是重複那幾首，沒有用過另外一個，蠻想試試看的

**Comment by dandanwu377** [comment]
我自己比較喜歡 Spotify，因為它的推薦系統很好，常常能挖到喜歡的新歌，介面也比較直覺、容易操作。不過如果是用 iPhone 或 Mac 的話，Apple Music 也很方便，能跟裝置無縫整合，音質也比較好。兩個平台各有優點，主要還是看你重視音質還是使用體驗。我有朋友為了空間音訊選 Apple Music，但也有人為了聽歌單和社群功能選 Spotify

**Comment by roy79231** [comment]
我會比較喜歡spotify，只是免費仔聽到一半吃廣告很燥就是了

**Comment by brianou583** [comment]
Youtube Music還是讚，訂閱Youtube Premium就可以同時享有無廣告的youtube還有和Spotify、Apple Music一樣功能的音樂軟體

**Comment by 77JoyWang77** [comment]
我只用過apple music，因為家庭專案，不知道其他播放軟體怎麼樣，但apple music裡面有的其實挺侷限的，老一點或太新的都有很大概率沒有

**Comment by LINSHIHMIN** [comment]
我自己一直以來都是用spotify，不曉得跟其他音樂平台比有什麼優缺點。因為喜歡的音樂通常都有上，而且如果想聽什麼演唱會歌單之類的，在搜尋查一下也幾乎都能看見有人已經創建了，想離線聽也可以先下載下來（只不過下載了就刪不掉的樣子），還可以跟朋友創一個即時聆聽的歌單，可以同時間一起聽一樣的歌，覺得使用下來沒什麼不便，所以也沒想過要使用其他app，就一直用著spotify。

**Comment by kyc3318** [comment]
spotify 100%

**Comment by mattwang1222** [comment]
spotify聽久了歌都一樣，apple的比較會推薦
